apps/evans/components/core/AudioPlayer.tsx

```diff
@@ -20,6 +20,27 @@ const AudioPlayer: React.FC<Props> = ({ tracks }) => {
   const [audio, setAudio] = useState<HTMLAudioElement | null>(null);
   const track = tracks[currentTrackIndex];
 
+  useEffect(() => {
+    const trackStorageItem = localStorage.getItem(window.location.pathname);
+    if (trackStorageItem) {
+      let {
+        storedTime,
+        storedTrackIndex,
+      }: { storedTime: number; storedTrackIndex: number } = JSON.parse(trackStorageItem);
+      if (storedTrackIndex >= tracks.length) {
+        storedTrackIndex = tracks.length - 1;
```

Comment:
i sort of lean towards saying that if the stored track index is too large, to reset to `0` instead of the last item